lib/driver.js

```diff
@@ -264,8 +264,13 @@ class XCUITestDriver extends BaseDriver {
     this.logEvent('appConfigured');
 
     // fail very early if the app doesn't actually exist
+    // or if bundle id doesn't point to an installed app
     if (this.opts.app) {
       await checkAppPresent(this.opts.app);
+    } else if (this.opts.bundleId) {
+      if (!await this.opts.device.isAppInstalled(this.opts.bundleId)) {
```

Comment:
That's why I added an alias for `device.isAppInstalled` to point to `device.isInstalled` in `lib/real-device-management.js`.